traitlets/traitlets.py

```diff
@@ -311,15 +311,25 @@ class that does the following:
     name = None
     this_class = None
 
+    def top_instance_init(self, obj, **kwargs):
```

Comment:
It seemed like these two PRs were part of the same thing, but I guess that's not how you see it? Should I ignore this one until we have reached a decision on #28?